streams/src/test/java/org/apache/kafka/streams/integration/utils/KafkaEmbedded.java

```diff
@@ -170,25 +174,47 @@ public void createTopic(final String topic,
                             final int replication,
                             final Properties topicConfig) {
         log.debug("Creating topic { name: {}, partitions: {}, replication: {}, config: {} }",
-            topic, partitions, replication, topicConfig);
-        try (KafkaZkClient kafkaZkClient = createZkClient()) {
-            final AdminZkClient adminZkClient = new AdminZkClient(kafkaZkClient);
-            adminZkClient.createTopic(topic, partitions, replication, topicConfig, RackAwareMode.Enforced$.MODULE$);
+                 topic, partitions, replication, topicConfig);
+        NewTopic newTopic = new NewTopic(topic, partitions, (short) replication);
+        newTopic.configs((Map) topicConfig);
+
+        int tries = 0;
+        int maxTries = 5;
+        try (AdminClient adminClient = createAdminClient()) {
+            while (true) {
+                try {
+                    adminClient.createTopics(Collections.singletonList(newTopic)).all().get();
+                    break;
+                } catch (InterruptedException | ExecutionException e) {
+                    if (tries++ > maxTries)
+                        throw new RuntimeException(e);
+
+                    try {
+                        Thread.sleep(100);
+                    } catch (InterruptedException e1) { }
+                }
+            }
         }
     }
 
-    private KafkaZkClient createZkClient() {
-        return KafkaZkClient.apply(zookeeperConnect(), false, DEFAULT_ZK_SESSION_TIMEOUT_MS,
-                DEFAULT_ZK_CONNECTION_TIMEOUT_MS, Integer.MAX_VALUE, Time.SYSTEM, "testMetricGroup", "testMetricType");
+    private AdminClient createAdminClient() {
```

Comment:
Seems to be redundant with the method in `EmbeddedKafkaCluster.java` -- might be better to make this method public and call within `EmbeddedKafkaCluster.java` ?